packages/block-library/src/query-pagination-next/edit.js

```diff
@@ -1,22 +1,86 @@
 /**
  * WordPress dependencies
  */
-import { __ } from '@wordpress/i18n';
-import { useBlockProps, PlainText } from '@wordpress/block-editor';
+import { __, _x } from '@wordpress/i18n';
+import {
+	useBlockProps,
+	PlainText,
+	InspectorControls,
+} from '@wordpress/block-editor';
+import {
+	PanelBody,
+	__experimentalSegmentedControl as SegmentedControl,
+	__experimentalSegmentedControlOption as SegmentedControlOption,
+} from '@wordpress/components';
 
 export default function QueryPaginationNextEdit( {
-	attributes: { label },
+	attributes: { label, arrow },
 	setAttributes,
 } ) {
 	return (
-		<PlainText
-			__experimentalVersion={ 2 }
-			tagName="a"
-			aria-label={ __( 'Next page link' ) }
-			placeholder={ __( 'Next Page' ) }
-			value={ label }
-			onChange={ ( newLabel ) => setAttributes( { label: newLabel } ) }
-			{ ...useBlockProps() }
-		/>
+		<>
+			<InspectorControls>
+				<PanelBody title={ __( 'Arrow settings' ) }>
```

Comment:
Do we need a panel if it's the only setting we currently have?

Reply:
We probably need it because there are other panels visible from `supports`, which render above this control. This is subjective though..

Reply:
Good point. Maybe this one should be just called "Settings".

Reply:
I've left as is for now and will update in a follow up when this lands: https://github.com/WordPress/gutenberg/pull/34017, which adds an extra visual label for `Segmented Control`. IMO it's not clear enough when having `Settings` as Panel title and a plain segmented control with some options.

Reply:
We can also call it "Arrow".